apps/website/src/components/layout/footer/Footer.tsx

```diff
@@ -12,22 +16,41 @@ export const Footer: React.FC = () => {
           <ul className="flex flex-wrap items-center justify-between">
             <li className="basis-full p-2 md:basis-1/3">
               <p>
-                Built by the Alveus.gg team and community,
-                <br />
+                Original design by{" "}
+                <Link
+                  href="https://chanelrooh.com"
+                  target="_blank"
+                  rel="noreferrer"
+                  className={footerLinkClasses}
+                >
+                  chanelrooh.com
+                </Link>
+                ,<br />
```

Comment:
```suggestion
                {", "}
                <br />
```
Similar nitpick -- also adding a space, so the text copies correctly